digitalin.py

```diff
@@ -56,7 +65,11 @@ def load_xml(self, node):
                     
                     else:
                         self.logger.warning(f"Unrecognized XML tag \'{child.tag}\' in <{self.expectedRoot}>")
-                    
+                            
+                except ValueError:
+                    self.logger.exception()
```

Comment:
logger.exception requires a message argument. Same comment for all other instances of this.